Hercules-DJControl-Inpulse-200-script.js

```diff
@@ -0,0 +1,249 @@
+
+// How much faster seeking (shift+scratch) is than scratching.
+DJCi200.scratchShiftMultiplier = 4;
+
+// How fast bending is.
+DJCi200.bendScale = 1.0;
+
+// DJControl_Inpulse_200_script.js
+//
+// ***************************************************************************
+// * Mixxx mapping script file for the Hercules DJControl Inpulse 200.
+// * Author: DJ Phatso, contributions by Kerrick Staley
+// *
+// * Version 1.2 (December 2019)
+// * Forum: https://www.mixxx.org/forums/viewtopic.php?f=7&t=12592
+// * Wiki: https://mixxx.org/wiki/doku.php/hercules_djcontrol_inpulse_200
+//
+// Changes to v1.2
+// - Code cleanup.
+//
+// Changes to v1.1
+// - Fix seek-to-start and cue-master behavior.
+// - Tweak scratch, seek, and bend behavior.
+// - Controller knob/slider values are queried on startup, so MIXXX is synced.
+// - Fixed vinyl button behavior the first time it's pressed.
+//
+// v1.0 : Original forum release				  
+//
+// TODO: Functions that could be implemented to the script:
+//
+//  FX:		
+//  - See how to preselect effects for a rack
+//  - Tweak scratch effect for Pad 4	
+//
+//************************************************************************
+//
+// We have to disable the no-unused-vars check because we have many MIDI
+// callbacks that receive a fixed list of arguments, but we usually don't use
+// most of these arguments. Eslint seems to make it relatively difficult to
+// disable this check on a case-by-case basis, so we disable it for the whole
+// file.
+// See this GitHub issue for more context:
+// https://github.com/eslint/eslint/issues/1939
+// *eslint-disable no-unused-vars*/
+
+DJCi200.kScratchActionNone = 0;
+DJCi200.kScratchActionScratch = 1;
+DJCi200.kScratchActionSeek = 2;
+DJCi200.kScratchActionBend = 3;
+
+//function DJCi200() {}
+
+
+DJCi200.init = function () {
+
+    DJCi200.scratchButtonState = true;
+    DJCi200.scratchAction = {
+        1: DJCi200.kScratchActionNone,
+        2: DJCi200.kScratchActionNone
+    };
+
+    //Turn On Vinyl button LED(one for each deck).
+    midi.sendShortMsg(0x91, 0x03, 0x7F);
+    midi.sendShortMsg(0x92, 0x03, 0x7F);
+
+    //Turn On Browser button LED
+    midi.sendShortMsg(0x90, 0x05, 0x10);
+
+    // Connect the Browser LEDs
+    engine.getValue("[Library]", "MoveFocus");
+    engine.getValue("[Master]", "maximize_library");
+
+    //Set effects Levels - Dry/Wet
+    engine.setParameter("[EffectRack1_EffectUnit1_Effect1]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit1_Effect2]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit1_Effect3]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect1]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect2]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit2_Effect3]", "meta", 0.6);
+    engine.setParameter("[EffectRack1_EffectUnit1]", "mix", 1);
+    engine.setParameter("[EffectRack1_EffectUnit2]", "mix", 1);
+
+
+

```

Comment:
I think a single newline suffices here (and also in other parts of the mapping).